### ISSUE TITLE
Can't login with default credentials

### ISSUE THREAD

**Issue by amityagov** [buildpal/buildpal#1]
Hi

Subject. Create docker container, tried to login with admin/admin123 and get message "Invalid user ID and/or password."

Thanks

**Comment by amityagov** [comment]
In Chrome Console i see error "403 (CORS Rejected - Invalid origin)".

**Comment by amityagov** [comment]
Also i tried to specify config.json as docker run argument and get 404 error.

**Comment by udaychandra** [comment]
I'm sure you must have done this but if you haven't already, can you check the docker logs by running "docker logs buildpal". Let's see if the logs shed some light. 
And just to confirm, you ran the buildpal docker image from docker hub and tried to login to the UI using the default creds, right?

**Comment by amityagov** [comment]
And tried to login from UI.

**Comment by amityagov** [comment]
I don't see any errors in logs. And yes, I run it from docker image.

**Comment by udaychandra** [comment]
I will try to reproduce the error. CORS should not come into the equation since the website and the REST API are hosted on a single vert.x JVM. And the [CORS ](https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/handler/impl/CorsHandlerImpl.java) code in vertx uses a simple java regex matcher for the whitelist.

**Comment by amityagov** [comment]
@udaychandra I ran it on virtual machine in docker. So host was not "localhost".

**Comment by udaychandra** [comment]
Got it! Vertx CORS, once enabled, starts matching the origin to the pattern even if the request came from the same domain. I pushed a new image with a pattern that matches any http or https request. Give it a shot when you get a chance.

I was also able to turn off CORS by specifying the config.json path in the run command - make sure that the file is available inside the docker container --I used a host mount (--mount type=bind,source="$(pwd)",target=/conf)
